engine/apps/alerts/escalation_snapshot/serializers/escalation_snapshot.py

```diff
@@ -8,11 +8,11 @@
 
 
 class EscalationSnapshotSerializer(serializers.Serializer):
-    channel_filter_snapshot = ChannelFilterSnapshotSerializer()
-    escalation_chain_snapshot = EscalationChainSnapshotSerializer()
+    channel_filter_snapshot = ChannelFilterSnapshotSerializer(allow_null=True, default=None)
+    escalation_chain_snapshot = EscalationChainSnapshotSerializer(allow_null=True, default=None)
     last_active_escalation_policy_order = serializers.IntegerField(allow_null=True, default=None)
-    escalation_policies_snapshots = EscalationPolicySnapshotSerializer(many=True)
-    slack_channel_id = serializers.CharField(allow_null=True)
+    escalation_policies_snapshots = EscalationPolicySnapshotSerializer(many=True, default=list)
+    slack_channel_id = serializers.CharField(allow_null=True, default=None)
```

Comment:
related to [this change](https://github.com/grafana/oncall/pull/1266/files#r1094955383)